usr/lib/common/sess_mgr.c

```diff
@@ -90,21 +90,23 @@ CK_RV session_mgr_new(CK_ULONG flags, CK_SLOT_ID slot_id,
     // we don't have to worry about having a user and SO session at the same
     // time. that is prevented in the login routine
     //
-    if (user_session) {
```

Comment:
I see you changed the transactional memory version and the locking version.
Makes sence as it is nearly impossible to just rework the locking code.
However, we should talk about if we want to keep the transactional memory code anyway - as this still means to
have two versions to maintain, test and verify.